docs/software/python-cli/index.mdx

```diff
@@ -543,6 +543,10 @@ meshtastic --ble-scan
 
 Don't start the API, just function as a dumb serial terminal. Probably not very helpful from the command line. Used more for testing/internal needs.
 
+:::note
```

Comment:
We're really trying to move away from using admonitions(e.g. note). What would you think of working this into the description itself?

Same comment for the second placement under `usage`

Reply:
oh, for sure! I just saw the existing admonitions and followed the style there. I'll rework.